core/templates/services/math-interactions.service.ts

```diff
@@ -62,7 +62,8 @@ export class MathInteractionsService {
       errorMessage = 'Your answer includes a division by zero, which is ' +
         'not valid.';
     }
-    if (errorMessage.indexOf('is not a valid postfix operator.') !== -1) {
+    if (
+      errorMessage.indexOf('is not a valid postfix operator.') !== -1) {
```

Comment:
Why this change, the line was only 76 chars.

Reply:
Fixed.